src/index.html

```diff
@@ -22,6 +26,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="container_logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="container_menu">
+        <ul class="menu">
+          <li>
+            <a
+              class="is-active"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__imac">
+          <img
+            class="card__imac-img"
+            src="../src/images/imac.jpeg"
+            alt="imac_photo"
+          />
+        </div>
+        <div class="card__info">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
```

Comment:
Avoid using the <br> tag for creating space between items. It's better to control spacing with CSS.